dev-java/gradle-bin/Manifest

```diff
@@ -1,2 +1,3 @@
-DIST gradle-3.3-all.zip 91388147 BLAKE2B 15539ec06cadcdaee9d54928381d5626bf06bae3251eb304fee095e6d0362f23759e99e3841dac2b9a1eed50b6057ecce78230dafb45711bfe764bf107b1f704 SHA512 4b852eaf4030c780516fe9214c289e49fc93fc62672da70b7fd6bbbcd128e6912dd196195ce32570984503ea53b48593a41fbf4ec69b0f18b5e32c2fe9e556b3
-DIST gradle-3.4.1-all.zip 91604607 BLAKE2B 1c4aaae4899f3faa94fc3ead84c4fd4bdb6212c0f579a255c7494896fd2a498890c58e853315892c3c9d78635b58da5423b1b2211b991639474d19eca1f2992f SHA512 bdaabf747e60ae1cf2c9fedb0730a2e3deb914609db05ffc7e4cbd5402bc44e84d2c98154f14fdd7f944c2f49eb5dead0f44c119ac5a16bd7491c3369d784f5c
+DIST gradle-4.9-all.zip 113472562 BLAKE2B a29303aab49eb79a9298fc1e7496ea2fc326846101ccbe5247a37b57c11b3e050ed65bb31a097912818cf0e5d8cbe0d2b20f8a4619f42011b696383e1578fd51 SHA512 e39ec89c2082e5d0c2f6c1d1f968045238056bf09108997b85e03b7493227c3d9d9c289f5b47e0c7aac5049885de89d22d3dcc3a4cd88a8d60229c0bfc95cb2b
+EBUILD gradle-bin-4.9.ebuild 1037 BLAKE2B 9c893331e327aa6ed65ff3b0158a313e67247c0498a0c4376e6c6e734b952ae7c8aab6e81f45d81debfea7bd5ab10cb8fe4b0ea1dfa8e7407bd4a51e9aa2da27 SHA512 44229dff167757e51bc0426a355ad5e40cd7f805ef4478868a484538f9a3c69cbad6cf85100cedd7bed4dfd0f9adf4b897358c466558cec2f30e803febba1a9c
```

Comment:
Use thin-manifests, that ::gentoo has by default. I've linked you docs needed to learn how to correctly fork ::gentoo repo.